.github/workflows/deepx-ci.yml

```diff
@@ -24,7 +24,7 @@ jobs:
           username: deepx-inc
           password: ${{ github.token }}
 
-      - name: Build and commit the Docker container
+      - name: Build
```

Comment:
Maybe `Build the Docker container`?

Reply:
I changed to build the docker image to be more accurate.